.gitlab-ci.yml

```diff
@@ -0,0 +1,22 @@
+stages:
+  - trigger
+
+
+trigger-veracode-scan:
+  # prepare and submit for static code analysis
+  stage: trigger
+  only:
```

Comment:
https://docs.gitlab.com/ce/ci/yaml/README.html#onlyrefsexceptrefs
Can also make sure this only runs on schedules using something like this if you like:
```
only:
    refs:
      - schedules
```

Reply:
It will only run if the variable is defined and schedules can be configured to define variables. So in the current setup, it will be equivalent except it can also be manually launched to test it works.

I've recommended using the variable approach instead of `only: schedules` in aktualizr repo as it's more powerful altogether.

Reply:
Ah cool. Yeah was just thinking it might be handy if ye didn't know about it, but ye've got it covered :)